src/txn/txn.c

```diff
@@ -572,11 +572,14 @@ __wt_txn_update_oldest(WT_SESSION_IMPL *session, uint32_t flags)
 
         /* Output a verbose message about long-running transactions,
          * but only when some progress is being made. */
+        current_id = __wt_atomic_loadv64(&txn_global->current);
+        WT_ASSERT(session, WT_TXNID_LE(oldest_id, current_id));
         if (WT_VERBOSE_ISSET(session, WT_VERB_TRANSACTION) &&
           current_id - oldest_id > (10 * WT_THOUSAND) && oldest_session != NULL) {
```

Comment:
I wonder how often this message will actually log as it i waiting for the gap to be > 10000. Probably not very often, if ever.

Reply:
hi, @luke-pearson 
Not very often, about a few outputs a second.